Source/EasyNetQ.DI.Tests/Properties/AssemblyInfo.cs

```diff
@@ -2,7 +2,7 @@
 
 // NOTE: Forcing xUnit to not run tests in parallel. This is because the
 // tests call RegisterAsEasyNetQContainerFactory which results in calling
-// static method RabbitHutch.SetContainerFactory.  As a result, the same
+// static method RabbitHutch.SetContainerFactory. As a result, the same
```

Comment:
@Pliner , @zidad , @micdenny I think this file is outdated. There are no `SetContainerFactory`/`RegisterAsEasyNetQContainerFactory` methods in the codebase.

Reply:
You are right.

Reply:
So? Keep it or delete? If keep then what to write in comments?

Reply:
Let's delete it. Thanks

Reply:
done